parser/parser.y

```diff
@@ -74,222 +74,223 @@ import (
-	restrict          "RESTRICT"
-	revoke            "REVOKE"
-	right             "RIGHT"
-	rlike             "RLIKE"
-	row               "ROW"
-	rows              "ROWS"
-	rowNumber         "ROW_NUMBER"
-	secondMicrosecond "SECOND_MICROSECOND"
-	selectKwd         "SELECT"
-	set               "SET"
-	show              "SHOW"
-	smallIntType      "SMALLINT"
-	spatial           "SPATIAL"
-	sql               "SQL"
-	sqlBigResult      "SQL_BIG_RESULT"
-	sqlCalcFoundRows  "SQL_CALC_FOUND_ROWS"
-	sqlSmallResult    "SQL_SMALL_RESULT"
-	ssl               "SSL"
-	starting          "STARTING"
-	statsExtended     "STATS_EXTENDED"
-	straightJoin      "STRAIGHT_JOIN"
-	tidbCurrentTSO    "TiDB_CURRENT_TSO"
-	tableKwd          "TABLE"
-	tableSample       "TABLESAMPLE"
-	stored            "STORED"
-	terminated        "TERMINATED"
-	then              "THEN"
-	tinyblobType      "TINYBLOB"
-	tinyIntType       "TINYINT"
-	tinytextType      "TINYTEXT"
-	to                "TO"
-	trailing          "TRAILING"
-	trigger           "TRIGGER"
-	trueKwd           "TRUE"
-	unique            "UNIQUE"
-	union             "UNION"
-	unlock            "UNLOCK"
-	unsigned          "UNSIGNED"
-	update            "UPDATE"
-	usage             "USAGE"
-	use               "USE"
-	using             "USING"
-	utcDate           "UTC_DATE"
-	utcTimestamp      "UTC_TIMESTAMP"
-	utcTime           "UTC_TIME"
-	values            "VALUES"
-	long              "LONG"
-	varcharType       "VARCHAR"
-	varcharacter      "VARCHARACTER"
-	varbinaryType     "VARBINARY"
-	varying           "VARYING"
-	virtual           "VIRTUAL"
-	when              "WHEN"
-	where             "WHERE"
-	write             "WRITE"
-	window            "WINDOW"
-	with              "WITH"
-	xor               "XOR"
-	yearMonth         "YEAR_MONTH"
-	zerofill          "ZEROFILL"
-	natural           "NATURAL"
+	add                  "ADD"
+	all                  "ALL"
+	alter                "ALTER"
+	analyze              "ANALYZE"
+	and                  "AND"
+	array                "ARRAY"
+	as                   "AS"
+	asc                  "ASC"
+	between              "BETWEEN"
+	bigIntType           "BIGINT"
+	binaryType           "BINARY"
+	blobType             "BLOB"
+	both                 "BOTH"
+	by                   "BY"
+	call                 "CALL"
+	cascade              "CASCADE"
+	caseKwd              "CASE"
+	change               "CHANGE"
+	character            "CHARACTER"
+	charType             "CHAR"
+	check                "CHECK"
+	collate              "COLLATE"
+	column               "COLUMN"
+	constraint           "CONSTRAINT"
+	convert              "CONVERT"
+	create               "CREATE"
+	cross                "CROSS"
+	cumeDist             "CUME_DIST"
+	currentDate          "CURRENT_DATE"
+	currentTime          "CURRENT_TIME"
+	currentTs            "CURRENT_TIMESTAMP"
+	currentUser          "CURRENT_USER"
+	currentRole          "CURRENT_ROLE"
+	currentResourceGroup "CURRENT_RESOURCE_GROUP"
```

Comment:
MySQL does not have `CURRENT_RESOURCE_GROUP` as a reserved keyword https://dev.mysql.com/doc/refman/8.0/en/keywords.html

Maybe we can make it non-keyword...
The different is that you can't `create table CURRENT_RESOURCE_GROUP (...)` after this change.

But the current way is still OK to me, since it follows the `CURRENT_XXX` as reserved keyword convention in MySQL

Reply:
Oh, yes, I think we can make it a non-keyword, so the compatibility should be better.